src/main/java/org/ua/javaPro/berezhnoy/hillelHomeWork16LambdaCSV/Main.java

```diff
@@ -0,0 +1,47 @@
+package org.ua.javaPro.berezhnoy.hillelHomeWork16LambdaCSV;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Main {
+
+    public static void main(String[] args) {
+        String line = "";
+        List<Hero> heroes = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader
+                ("src/main/resources/heroes_information.csv"))) {
+            bufferedReader.readLine();
+            while ((line = bufferedReader.readLine()) != null) {
+                String[] heroData = line.substring(line.indexOf(";") + 1).split(";");
```

Comment:
Це не одразу зрозуміло чому так робиться, тому краще ось так: 
```suggestion
                String[] heroData = line.split(";");
```
а далі просто додали +1 в кожному доступі по індексу: 
```new Hero(heroData[1], heroData[2], heroData[3], ```